README.md

```diff
@@ -2,7 +2,7 @@
 
 [![NCSA licensed](https://img.shields.io/badge/license-NCSA-blue.svg)](LICENSE)
 [![CI](https://github.com/ILLIXR/ILLIXR/workflows/illixr-tests-master/badge.svg)](https://github.com/ILLIXR/ILLIXR/actions)
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/ILLIXR/community)
+![Discord](https://img.shields.io/discord/830812443189444698)
```

Comment:
I think this might be missing a link to our actual Discord, would that be possible?

Reply:
I would need a Discord admin to get the server ID (as specified [here](https://github.com/dgibbs64/discord-banners/blob/master/README.md))

Reply:
I am not sure, but I just checked the server ID is 830812443189444698 which is the same as the one mentioned in this current version of documentation

Reply:
![Discord](https://discordapp.com/api/guilds/830812443189444698/widget.png?style=banner2)
so something like this?

Reply:
does this work?
`![Discord](https://discordapp.com/api/guilds/830812443189444698/widget.png?style=shield)`

Reply:
I think you might have to enable "widget" on Discord ([ref](https://github.com/FriiZoLoGYy/Discord-Github-Badges))

Reply:
I just enabled server widget. Give it a try now